docs/lib/Components/index.js

```diff
@@ -125,7 +125,7 @@ class Components extends React.Component {
       <Container fluid className="content">
         <Row>
           <Col md={{ size: 3, push: 9 }}>
-            <div className="docs-sidebar mb-3">
+            <div className="docs-sidebar m-b-3">
```

Comment:
Isn't this the old syntax? Latest docs show the following as patterns for utility spacing classes:

> The classes are named using the format {property}{sides}-{size} for xs and {property}{sides}-{breakpoint}-{size} for sm, md, lg, and xl.